### DISCUSSION TITLE
Is it possible to hide customer names to regular users?

### DISCUSSION THREAD

**Discussion by ghost** [kimai/kimai#3687] [Questions & Answers] [answered]
_Note: I already used the search to find an answer, but I didn't find any._

Hello everyone,

I just created a new user and added him to a team (this team has as project TPSW). This new user has the role ROLE_USER and I didn't change anything in the permissions settings.

I expect that when creating a time booking, the user only sees the clients that are linked to the project. But he sees all customers.

What am I doing wrong?

My team settings:

![grafik](https://user-images.githubusercontent.com/51318612/208104700-3c25e4ef-bc2a-4c39-8c6b-de27cfcf55dc.png)

Best,
Niklas

**Comment by kevinpapst** [comment, marked as answer]
Read the documentation. Items (here customer) are visible, unless you assign teams to them.

**Reply by ghost** [reply]
Thank you. Didn't figured that out by reading the documentation, to be honest, I'm always having a look on the documentation first.

**Reply by kevinpapst** [reply]
Important feedback, thank you 👍 

Maybe I have to add cross-references... can you help me improving the docs please.

https://www.kimai.org/documentation/teams.html says 

> Customers - access can be limited by assigning one or more teams. Customers without any team assignment can be seen by every user. Projects for this customer will be limited as well, if a team is assigned
> Projects - access can be limited by assigning at least one team. Projects without any team assignment can be seen by every user, which has access to the related customer

Where did you look for info? 
And where would you expect the info which you asked for?

**Reply by ghost** [reply]
Many thanks, this is the information I should've found and it's already pretty good documented and at the right place as well (IMHO).

My mistake was to not look into the teams section of the documentation. I thought it was an permission issue (maybe because of the brain of a webdev) and so I had a look on the https://www.kimai.org/documentation/permissions.html page and tried a bit (created custom user role, assigned and removed a couple of permissions, et cetera).

Maybe you could add a text box below the introduction text to the page "Roles & Permissions" which says something like: "Are you looking for the assignment/linking(?) of teams or projects to Roles/Permissions? Please check https://www.kimai.org/documentation/teams.html OR https://www.kimai.org/documentation/project.html."

(Just want to use this little chat for positive feedback: the documentation of Kimai2 is one of the most extensive and well-structured I've ever seen. That's very cool! I know that Kimai2 is a passion project and how much time and effort it takes to take care of a project like this.)

Hope my thoughts help you. Otherwise feel free to reach out, I'm glad when I can contribute to Kimai2!

**Reply by kevinpapst** [reply]
So you mean that this sentence should be highlighted somehow, so people do not mis it?

<img width="772" alt="Bildschirm­foto 2022-12-16 um 18 50 21" src="https://user-images.githubusercontent.com/533162/208158558-4d670cb4-1d74-447e-8cff-68f4dd007721.png">

**Reply by ghost** [reply]
Right, that would be a possible solution. Like you did at page "Teams" maybe.

![grafik](https://user-images.githubusercontent.com/51318612/208246957-f173b5b1-04e8-4572-8633-9a46ce9455cc.png)

**Reply by kevinpapst** [reply]
Done 👍
